packages/react-core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -32,6 +32,8 @@ export interface DropdownProps extends MenuProps, OUIAProps {
   zIndex?: number;
   /** Additional properties to pass to the Popper */
   popperProps?: Partial<PopperProps>;
+  /** Keys that trigger onOpenChange, defaults to tab and escape. */
```

Comment:
I lean towards the flexibility personally, but if we rather limit what a user can press to trigger `onOpenChange` I can update the prop. I do vote to keep the prop named as is to allow for more or less keys as needed in the future (maybe as an object with `Tab`/`Escape` as bool fields) rather than have a separate prop just for `tab` though. LMK